adoc/admin-updates.adoc

```diff
@@ -17,6 +17,23 @@ Run `sudo zypper update` on the management workstation before any attempt to upd
 
 Updating of {kube} components is handled via `skuba`.
 
+The general procedure should look like this:
+
+. Update the packages and reboot your management workstation to get all the latest changes to skuba and it's dependencies
+. Run `skuba addon upgrade apply` on the management workstation
```

Comment:
first we need "addon upgrade plan" and then "apply".
After this we need to re-apply the config changes done in any addon before the upgrade apply. (upgrade apply i.e. removes the ldap settings done in dex.yaml).